tox.ini

```diff
@@ -41,7 +40,6 @@ skip_install = true
 sitepackages = {env:SITE_PACKAGES:false}
 passenv =
     HOME
-    EXTRA_DEPS
```

Comment:
IIRC `EXTRA_DEPS` is used by `hdev requirements` to install `pip-tools` into the venv. This is no longer necessary since this PR adds `pip-tools` to tox's `deps` anyway. Also it was never necessary in the first place: `pip-compile` needs to be run with the right version of Python but doesn't need to be run in the venv. `hdev requirements` can just install `pip-tools` in pyenv and run `pip-compile` from pyenv (`PYENV_VERSION=X.Y.Z pyenv exec pip install pip-tools && PYENV_VERSION=X.Y.Z pyenv exec pip-compile ...`).